mini-e2e-helm.groovy

```diff
@@ -206,7 +206,8 @@ node('cico-workspace') {
 				} else if ("${test_type}" == "nfs"){
 					test_type = '--test-nfs=true --test-cephfs=false --test-rbd=false'
 				}
-				ssh "cd /opt/build/go/src/github.com/ceph/ceph-csi && make run-e2e NAMESPACE='${namespace}' E2E_ARGS=\"--delete-namespace-on-failure=false --deploy-cephfs=false --deploy-rbd=false --helm-test=true ${test_type}\""
+				def e2e_args = "--delete-namespace-on-failure=false --deploy-cephfs=false --deploy-rbd=false --helm-test=true ${test_type}"
+				ssh "cd /opt/build/go/src/github.com/ceph/ceph-csi && make run-e2e NAMESPACE='${namespace}' E2E_ARGS=\"${e2e_args}\""
```

Comment:
you're mixing two notations here:
```
NAMESPACE='${namespace}'
```
and
```
E2E_ARGS=\"${e2e_args}\"
```

My preference goes to the 1st, as the escaping of `"` is a little ugly.

Reply:
Also note that Groovy has two ways of creating strings:

1. using `'single quotes'`, where variables like `${test_type}` will not be replaces
2. using `"double quotes"`, where variables will be replaced

Escaping of `'` in a `"double quoted"` string is not needed, making this rather clean:
```groovy
ssh "cd ... && make run-e2e NAMESPACE='${namespace}' E2E_ARGS='${e2e_args}'"
```

Reply:
I am a little skepticle on this option since the previous approach of escaping `"double qouted"` string did not work as intended. Lets try this. Hopefully it should work.

Reply:
Done.